spec/support/factories.rb

```diff
@@ -4,6 +4,10 @@
     password "password"
     password_confirmation "password"
     role "default"
+
+    factory :user_with_orders do
```

Comment:
This is a great idea. We probably need to add more combined factories to help DRY up our test suite.